pkg/okteto/context.go

```diff
@@ -191,7 +191,10 @@ func K8sContextToOktetoUrl(ctx context.Context, k8sContext, k8sNamespace string,
 
 	// check the namespace label
 	if k8sNamespace == "" {
-		k8sNamespace = cfg.Contexts[k8sContext].Namespace
+		cfgK8sNamespace, exists := cfg.Contexts[k8sContext]
+		if exists {
+			k8sNamespace = cfgK8sNamespace.Namespace
+		}
```

Comment:
```suggestion
		} else{
		return k8sContext
		}
```

can we add here the else condition to return what is checked on line 200-202. If k8sNamespace still is empty would fall into this case. Instead of checking the same condition twice at 200 and 193 ? WDYT?